charts/atlantis/stages/production-terraform-cloud/initial_setup_values.yaml

```diff
@@ -0,0 +1,9 @@
+# Use this values file when perfoming the initial registration of the GitHub App.
+# https://www.runatlantis.io/docs/access-credentials.html#generating-an-access-token
+# Use the values.yaml file after you've completed initial setup.
+github:
+  user: fake
+  token: fake
+
+environment:
+  ATLANTIS_ATLANTIS_URL: http://atlantis-terraform-cloud.production.procoretech.com/
```

Comment:
Any plans to make DNS entry for this and enable UI for Atlantis for TFCloud?

Reply:
I'll probably have to speak with Traffic with that. We can log a discovery ticket for this.

Reply:
Can ingress gateway be configured to allow specific traffic routes only from VPN range?

Reply:
I think that can be gated from the Load Balancer, rather than Ingress.